### PR TITLE
Added description to README.md

### LINKED ISSUE #24: Add description of what Maestro is to the README
As we start to get more people involved in hackEDU, I believe it's important to clearly convey what Maestro does, especially considering it's a part of our curriculum now.

### PR DESCRIPTION
**Pull request by TheThirdOne**
Fixes #24

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -9,6 +9,50 @@ Authors: [@TheThirdOne](https://github.com/TheThirdOne),
...
 ![](stark.gif)
 
+-------------------------------------------------------------------------------
+
+## Objective
+
+The objective of Maestro is to allow a beginner to use services like 
+[Twilio](https://www.twilio.com/products) without being slowed by the details
+of how to implement the API. This will help beginners build more/better 
+applications which will help encourage them to continue learning.
```
Comment (zachlatta) [inline README.md]: How about mentioning that it makes it possible for people to use Twilio straight from their frontend JavaScript?

File: README.md [hunk 2]
```diff
@@ -9,6 +9,50 @@ Authors: [@TheThirdOne](https://github.com/TheThirdOne),
...
+
+## Usage
+
+The JS file containing the library is found at `examplehost.com/static/baton.js`
```
Comment (zachlatta) [inline README.md]: Perhaps just say `/static/baton.js`? I imagine `examplehost.com` may be a bit confusing.

File: README.md [hunk 3]
```diff
@@ -9,6 +9,50 @@ Authors: [@TheThirdOne](https://github.com/TheThirdOne),
...
+    - SMS and calling
+    - Wraps both the Twilio [Restful API](https://www.twilio.com/docs/api/rest) 
+    and the [TWIML API](https://www.twilio.com/docs/api/twiml)
+  - [Giphy](https://github.com/Giphy/GiphyAPI):
+    - Image search
+  - Echo:
+    - A simple module to help serve as template for module developers
+  - [Neutrino api](https://www.neutrinoapi.com/api/):
+    - Currently unfinished
+    - The most useful commands would be Geocode Address and Geocode Reverse
```
Comment (zachlatta) [inline README.md]: Can you update this to follow our markdown style guide? https://github.com/hackedu/meta/blob/master/markdown_style_guide.md (no 2 spaces before bullet points)

File: README.md [hunk 4]
```diff
@@ -9,6 +9,50 @@ Authors: [@TheThirdOne](https://github.com/TheThirdOne),
...
+### Examples
+
+```
+maestro.Twilio.recieveSms("*",function(e){ //recieve any incoming sms
+  maestro.Twilio.makeCall(e.From,e.To,maestro.Twilio.twiml().play(e.Body)); //call back  playing any sound file in the message
+  console.log("Calling",e.From,"from",e.To,"playing",e.Body)
+});
+```
+
+More examples can be found in the [Docs](docs.md).
```
Comment (zachlatta) [inline README.md]: Should "Docs" be capitalized?

File: README.md [hunk 5]
```diff
@@ -9,6 +9,50 @@ Authors: [@TheThirdOne](https://github.com/TheThirdOne),
...
+Open up the javascript console and [try it out!](http://maestro.ngrok.com/static/)
+
+### Examples
+
+```
+maestro.Twilio.recieveSms("*",function(e){ //recieve any incoming sms
+  maestro.Twilio.makeCall(e.From,e.To,maestro.Twilio.twiml().play(e.Body)); //call back  playing any sound file in the message
+  console.log("Calling",e.From,"from",e.To,"playing",e.Body)
+});
+```
```
Comment (zachlatta) [inline README.md:38]: Perhaps this code would be easier to read if function arguments were separated by spaces?

**Comment by TheThirdOne** [comment]
Is this ready to merge?

**Comment by zachlatta** [comment]
Indeed!

**Pull request merged**